pandas/core/tools/numeric.py

```diff
@@ -19,6 +19,14 @@ def to_numeric(arg, errors='raise', downcast=None):
     depending on the data supplied. Use the `downcast` parameter
     to obtain other dtypes.
 
+    Please note that precision loss may occur if really large numbers
+    are passed in. Due to the internal limitations of `ndarray`, if
+    numbers smaller than `-9223372036854775808` or larger than
```

Comment:
can you also show the expr for these ,e g. -np.info(np.int64).max and corresponding for uint64 max

Reply:
Done.